### PR TITLE
draft: initiate french translation

### PR DESCRIPTION
**Pull request by cuitlazotlac**
## Summary of changes

_Description of the changes included in this pull request. 
Steps to follow to confirm the feature is completed or the bug/issue is resolved. 
Are there any dependencies? 
Does this code break any existing functionality or introduce any risk?_

## Relevant Context

- Feature: [ticket number](link to project management ticket)
- Relevant Pull Request: # 

## Further comments

--- 
## Types of changes

- [x] Content edit, File upload, content addition
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Technical Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: content/fr/UI.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "Bug_Report": "Signaler un bug",
+  "Go_Back": "Revenir en arrière",
+  "Language": "French",
+  "Flag": "🇫🇷",
+  "Search": "Rechercher",
+  "No_Results": "Aucun résultat '{{query}}'",
+  "Home": "Home",
```
Comment (anthonygraignic) [inline content/fr/UI.json:8]: On peut aussi le traduire par "Accueil" non ?

File: content/fr/UI.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  "Previous": "Précédent",
+  "Next": "Suivant",
+  "Page_Language_Selector": "Page de séléction des langues",
+  "Available_Languages": "Langues disponible",
```
Comment (anthonygraignic) [inline content/fr/UI.json:12]: Langues disponibles

File: content/fr/UI.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  "LANGUAGES": "LANGUES",
+  "Available_Languages_none": "Pas de langues disponibles",
```
Comment (anthonygraignic) [inline content/fr/UI.json:14]: Ou "Aucune langue disponible"

File: content/fr/UI.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  "Need_Another_Language": "Besoin d'une autre langue ?",
+  "Join_translation_team": "Rejoindre l'équipe de traduction",
+  "Translations": "Traductions",
+  "Back_To_Main_Menu": "Recherche à la page principale",
```
Comment (anthonygraignic) [inline content/fr/UI.json:18]: Plutôt "Retourner au menu principal" ?

File: content/fr/UI.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  "aria_FooterLogo": "Kernel Footer Logo",
+  "aria_GithubIcon": "Github",
+  "aria_TwitterIcon": "Twitter",
+  "aria_YoutubeIcon": "Youtube",
+  "aria_ExternalWebsite": "Site web externe",
+  "kernel_icon": "Kernel",
+  "Chat": "Chat",
+  "Love": "Amour",
+  "NotFoundPage": {
+    "seoTitle": "404 Page Non Trouvé",
```
Comment (anthonygraignic) [inline content/fr/UI.json:28]: "404 Page Introuvable" ou "404 Page Non Trouvée"

File: content/fr/UI.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    "line_1": "Uh oh, la page que vous recherché n'existe pas.",
```
Comment (anthonygraignic) [inline content/fr/UI.json:29]: "Uh oh, la page que vous recherchez n'existe pas."

File: content/fr/UI.json [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    "line_2": "Nous avons déplacer quelques petites choses récemment.",
```
Comment (anthonygraignic) [inline content/fr/UI.json:30]: "Nous avons déplacé quelques petites choses récemment."

File: content/fr/UI.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    "line_3": "Vous pensez que nous avons un problème de notre coté ?"
+  },
+  "Footer": {
+    "Contribute": "Contribuer",
+    "Contact": "Contact",
+    "Blog": "Blog",
+    "Code": "Code",
+    "Caution": "Prudence",
+    "Cogitate": "Méditer",
```
Comment (anthonygraignic) [inline content/fr/UI.json:39]: Ou "Penser" ?

File: content/fr/UI.json [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    "Conceal": "Dissimuler",
+    "About": "A propos",
+    "Principles": "Principes",
+    "Start": "Commencer",
+    "What is Kernel?": "Kernel, c'est quoi ?",
```
Comment (anthonygraignic) [inline content/fr/UI.json:44]: Plutôt "Qu'est-ce Kernel ?"

File: content/fr/learn/module-0/index.mdx [hunk 10]
```diff
@@ -0,0 +1,92 @@
+---
+title: Introduction a Kernel
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/index.mdx:2]: "Introduction à Kernel"

File: content/fr/learn/module-0/play-of-pattern.mdx [hunk 11]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/play-of-pattern.mdx:1]: This file is not in French.

File: content/fr/learn/module-0/index.mdx [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+order: 0
+hideLanguageSelector: false
+description: Une description générale du parcours d'apprentissage Kernel qui vous apprendra à penser et à créer un meilleur web.
+featuredImage: images/shares/play-of-pattern.png
+---
+
+# 🌠 Module 0 - Une introduction à Kernel
+
+Kernel est un cours de 8 semaines où nous couvrirons un nouveau module d'apprentissage chaque semaine.
+Chaque module est composé de deux sections : **Les Lectures Artisanales**, que vous devez avoir lues avant
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/index.mdx:12]: **Lectures préparées** plutôt ? Artisanales c'est bizarre

File: content/fr/learn/module-0/index.mdx [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+le début de chaque semaine; et **Matériel Sélectionné** qui est composé d'articles, de vidéos et parfois
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/index.mdx:13]: plus **Ressources sélectionnées"

File: content/fr/learn/module-0/index.mdx [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+Basé sur le travail que nous couvrirons dans le module 3, Kernel vise à présenter l'apprentissage dans un nouveau type de support qui intègre un système de mémoire à répétition espacée. Cela signifie que vous devrez revenir pour faire des révisions ultérieures. Dans un cours conventionnel, vous oublierez la plupart de ce que vous avez appris au fil du temps, ne retenant peut-être qu'une poignée d'idées. Cependant, avec la répétition espacée intégrée au support, un petit engagement supplémentaire de temps qui signifie que vous vous souviendrez de beaucoup plus de choses. Faire cela ne sera pas difficile, ce sera plus facile que la lecture initiale. En outre, vous serez en mesure de lire d'autres documents qui s'appuient sur ces idées. Cela vous ouvrira le champ des possibilités.
+
+Veuillez nous faire plaisir en répondant aux questions que vous trouvez dans le texte. Pour chaque question, réfléchissez à ce que vous pensez être la réponse, cliquez pour révéler la réponse réelle, puis indiquez si vous vous en souvenez ou non. Si vous vous en souvenez, c'est très bien. Si vous ne vous en souvenez pas, c'est aussi bien, notez simplement la bonne réponse et continuez.
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/index.mdx:27]: "Merci de répondre aux questions que vous trouverez au fil du module."

File: content/fr/learn/module-0/index.mdx [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+Cette approche de répétition espacée explique pourquoi les questions ne demandent que quelques secondes de lecture et de réponse. Il ne s'agit pas d'exercices complexes, à la manière d'un manuel scolaire. Les questions ont plutôt un objectif différent : la promesse faite par chaque question est que vous vous souviendrez de la réponse pour toujours. Il s'agit de changer définitivement votre façon de penser.
+
+<Flash>
+
+<Card>
+
+Q: Quel est mon objectif
+
+A: Pour changer votre façon de penser ! (Et pour faire passer le beurre, bien sûr)
```
Comment (anthonygraignic) [inline content/fr/learn/module-0/index.mdx:37]: "Changer de façon de penser !"

**Review by anthonygraignic — CHANGES_REQUESTED** [review]
That's great ! Thank you, I suggested a few changes as I am a native speaker.
Also there is a file in spanish to remove :)

**Comment by andytudhope** [comment]
Approved and merged in #334 - thank you very much @cuitlazotlac :heart: :pray:

**Pull request closed without merging**